.eslintrc.js

```diff
@@ -22,7 +24,11 @@ module.exports = {
         'plugin:unicorn/recommended',
     ],
     rules: {
-        // 'simple-import-sort/imports': 'error',
+        'simple-import-sort/imports': 'error',
+        'simple-import-sort/exports': 'error',
+        'sort-imports': 'off',
+        'import/order': 'off',
+        "unused-imports/no-unused-imports": "error",
```

Comment:
Could we have single quotes instead of double quotes to keep it consistent?

Reply:
done!